### DISCUSSION TITLE
Enable schema validation

### DISCUSSION THREAD

**Discussion by fab1aan** [quarkiverse/quarkus-cxf#449] [Q&A] [answered]
I'm trying to migrate a webservice to quarkus. I want schema validation to be enabled. Until now, it was solved like this in cxf-servlet.xml:

```
<jaxws:endpoint name="asdfService" implementor="com.webservice.impl.asdfService"
wsdlLocation="/wsdl/asdfService.wsdl" address="/asdfService">
	<jaxws:properties>
		<entry key="schema-validation-enabled" value="true"/>
	</jaxws:properties>
</jaxws:endpoint>
```

How is this done with quarkus-cxf? Thanks for your help!

**Comment by shumonsharif** [comment, marked as answer]
Hi @fab1aan You can use the `org.apache.cxf.annotations.SchemaValidation` annotation on your implementor class.  Further details of the various types of schema validation are here:

https://cxf.apache.org/docs/annotations.html
